packages/resolver/lib/sources/fs.ts

```diff
@@ -65,18 +65,22 @@ export class FS implements ResolverSource {
     ];
 
     let body, filePath;
-    possiblePaths.forEach(possiblePath => {
+    for (let i = 0; i < possiblePaths.length; i++) {
       try {
+        const possiblePath = possiblePaths[i];
```

Comment:
Looks like we can just use `for (const ... of ...)` syntax here (it's cleaner)

```suggestion
    for (const possiblePath of possiblePaths) {
      try {
```